zerver/lib/actions.py

```diff
@@ -1185,6 +1261,7 @@ def notify_subscriptions_added(user_profile, sub_pairs, stream_emails, no_log=Fa
     send_event(event, [user_profile.id])
 
 def bulk_add_subscriptions(streams, users):
+    # type: (Iterable[Stream], Iterable[UserProfile]) -> Tuple[List[Tuple[UserProfile, Stream]], List[Tuple[UserProfile, Stream]]]
```

Comment:
Aren't the tuples in the return value (UserProfile, stream_name), not (UserProfile, Stream)?